lib/client/api_test.go

```diff
@@ -1365,3 +1375,11 @@ func TestGetTargetNodes(t *testing.T) {
 		})
 	}
 }
+
+func TestNonRetryableError(t *testing.T) {
+	err := NewNonRetryableError(trace.AccessDenied("do not enter"))
+	require.Error(t, err)
+	require.Equal(t, "do not enter", err.Error())
+	require.True(t, isNonRetryableError(err))
+	require.True(t, trace.IsAccessDenied(err))
```

Comment:
Keep going:

```suggestion
	assert.Equal(t, "do not enter", err.Error())
	assert.True(t, isNonRetryableError(err))
	assert.True(t, trace.IsAccessDenied(err))
```

Also please add error messages to the assertions.

* https://go.dev/wiki/TestComments#keep-going

Reply:
updated. we should do this more often in our repo.

Reply:
For sure.